### PR TITLE
SQS integration tests

### PR DESCRIPTION
**Pull request by georgepsarakis**
## Description

Enables integration tests with the SQS broker.

- [x] Update `.travis.yml` and `tox.ini` files.
- [x] Add CI script that starts [SQS moto server](https://github.com/spulec/moto).
- [x] Skip Manager operations that would not work with SQS.

### CODE REVIEW

**Comment by georgepsarakis** [comment]
@auvipy @thedrow although this generally is working, I am seeing some failures intermittently and I do not understand what may be wrong. If you could offer some help I would appreciate it.

File: tox.ini [hunk 1]
```diff
@@ -49,6 +54,36 @@ basepython =
     pypy: pypy
     flake8,flakeplus,apicheck,linkcheck,configcheck,pydocstyle: python2.7
 
+[testenv:2.7-integration-sqs]
```
Comment (georgepsarakis) [inline tox.ini]: If anyone knows a way to rewrite these `tox` sections in a less repetitive way, please let me know.

**Comment by thedrow** [comment]
You might have encountered a bug in the SQS broker code or maybe the moto SQS server isn't responding fast enough.

**Comment by thedrow** [comment]
@georgepsarakis strange... check the test failures here: https://github.com/celery/celery/pull/3977

**Comment by georgepsarakis** [comment]
@thedrow yes it is strange. Do you think perhaps it is a combination of Canvas and the Redis result backend?

**Comment by thedrow** [comment]
It might be a combination of the implementation of the "half-async" brokers such as Redis and SQS. They are async only when consuming and not when publishing.

**Comment by thedrow** [comment]
@georgepsarakis Are you still working on this or should we turn this to an issue and close the PR?

**Comment by georgepsarakis** [comment]
@thedrow I will resume efforts at a later time, unfortunately I could not achieve much.

**Pull request closed without merging**